### PR TITLE
feat: Quest Rewards to Gold Table

### PR DESCRIPTION
**Pull request by antonyip**

### CODE REVIEW

File: data/defikingdoms_itemtogold/dfk_item_to_gold.csv [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+0x6e1bc01cc52d165b357c42042cf608159a2b81c1,1,12.5
+0x78aed65a2cc40c7d8b0df1554da60b38ad351432,1,3.0
+0x8bf4a0888451c6b5412bcad3d9da3dcf5c6ca7be,1,5.0
+0xb80a07e13240c31ec6dc0b5d72af79d461da3a70,1,50.0
+0xcdffe898e687e941b124dfb7d24983266492ef1d,1,75.0
+0xe4cfee5bf05cef3418da74cfb89727d8e4fee9fa,1,5.0
+0xac5c49ff7e813de1947dc74bbb1720c353079ac9,1,5.0
+0xc0214b37fcd01511e6283af5423cf24c96bb9808,1,12.5
+0x19b9f05cde7a61ab7aae5b0ed91aa62ff51cf881,1,10.0
+0xc5891912718ccffcc9732d1942ccd98d5934c2e1,1,15.0
```
Comment (chuxinh) [inline data/defikingdoms_itemtogold/dfk_item_to_gold.csv]: add a line 
where's the source of this csv?

Comment (antonyip) [reply data/defikingdoms_itemtogold/dfk_item_to_gold.csv]: Gooood question.. lol… let me back track.

File: data/defikingdoms_itemtogold/readme.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+where evm_contract_address = '0xe53bf78f8b99b6d356f93f41afb9951168cca2c6'
+limit 10
+```
+
+NPC_VERSION column is an internal column I added for the future if DFK decides to modify the Swap prices.
+
+Should be able to do something like
+
+```
+CASE WHEN
```
Comment (chuxinh) [inline data/defikingdoms_itemtogold/readme.md]: lower case for the query

**Review by chuxinh — APPROVED** [review]
LGTM after nits

**Pull request merged**